.github/workflows/ci-sql-cli.yaml

```diff
@@ -26,6 +26,12 @@ concurrency:
   group: ${{ github.workflow }}-${{ github.event.pull_request.number || github.ref }}
   cancel-in-progress: true
 
+# This list should only have non-sensitive environment variables.
+# Environment variables with secrets should be in the specific jobs.
+env:
```

Comment:
Do we need to define these environment variables in the context of all tests - or are they only required for a specific test related to rendering `REDSHIFT` a connection file which has environment variables? If the latter, we can consider patching them only within the test of interest, using something like:
```
@mock.patch.dict(os.environ, {"REDSHIFT_DATABASE": "fake.redshift.amazonaws.com", ...}, clear=True)
```

Reply:
Right. Nice suggestion, thanks. Removed it from here.